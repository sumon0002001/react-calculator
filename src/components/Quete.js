import React from 'react';

const Quete = () => {
    return (
        <main>
        <div >
          <q>
            Mathematics is not about numbers, equations, computations, or algorithms:
            it is about understanding. â€“William Paul Thurston
          </q>
        </div>
      </main>

    )
}

export default Quete;