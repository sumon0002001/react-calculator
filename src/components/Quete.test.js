import React from 'react';
import renderer from 'react-test-renderer';
import Quete from './Quete';

it('Renders the Quote page', () => {
  const quete = renderer
    .create(<Quete />)
    .toJSON();
  expect(quete).toMatchSnapshot();
});
