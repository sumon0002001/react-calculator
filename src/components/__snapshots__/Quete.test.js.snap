// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Quote page 1`] = `
<main>
  <div>
    <q>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
    </q>
  </div>
</main>
`;
